index.html

```diff
@@ -28,6 +28,7 @@
 
 		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
 			<ul class="nav navbar-nav navbar-left">
+				<li class="nav-back" style="display:none;"><a>&#10094; View another chat</a></li>
```

Comment:
Instead of using unicode character, i think using a Glyphicon would be a better choice, since they come in bootstrap package.
Reference: https://getbootstrap.com/docs/3.3/components/#glyphicons